dkg/dkg.go

```diff
@@ -242,7 +290,7 @@ func aggSignLockHash(ctx context.Context, tcpNode host.Host, nodeIdx cluster.Nod
 	// Aggregate all group signatures.
 	b, err := tbls.Scheme().AggregateSignatures(sigs...)
 	if err != nil {
-		return nil, errors.Wrap(err, "aggregate signature")
+		return nil, errors.Wrap(err, "exchangeSigs signature")
```

Comment:
nit
```suggestion
		return nil, errors.Wrap(err, "exchange sigs signature")
```

Reply:
this got committed by mistake
reverting